.github/ISSUE_TEMPLATE/other.yml

```diff
@@ -0,0 +1,21 @@
+name: Other
+description: Use this for any other issues. Please do NOT create blank issues
+title: '[OTHER]'
+body:
+  - type: markdown
+    attributes:
+      value: '# Other issue'
+  - type: textarea
+    id: issuedescription
+    attributes:
+      label: What would you like to share?
+      description: Provide a clear and concise explanation of your issue.
+    validations:
+      required: true
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this issue?
+    validations:
+      required: false
```

Comment:
```suggestion
      required: false

```

Reply:
🤔 Hey could I know what's the issue over here ?

Reply:
End line

Reply:
> 

Is that for formatting or is it required ? 🤔 
I am new to making these issue templates via `.yml`, so just wanted to know